Gemfile

```diff
@@ -1,17 +1,3 @@
-# Copyright (c) 2014-2019 Cisco and/or its affiliates.
```

Comment:
This copyright notice is important. I know PDKSync strips it, we didn't figure out a way of automatedly adding it back, so it's a manual paste

Reply:
Have added the notice back